horizons/__init__.py

```diff
@@ -0,0 +1,281 @@
+        params['VECT_TABLE'] = None            # VEC: Table format
+        params['VECT_CORR']  = None            # VEC: correction level
+        params['CAL_FORMAT'] = cal_format # OBS: Type of date output (CAL, JD, BOTH)
+        params['ANG_FORMAT'] = 'HMS'           # OBS: Angle format (HMS or DEG)
+        params['APPARENT']   = None            # OBS: Apparent coord refract corr (AIRLESS or REFRACTED)
+        params['TIME_DIGITS'] = 'MINUTES'      # OBS: Precision (MINUTES, SECONDS, or FRACSEC)
+        params['TIME_ZONE']  = None            # Local civil time offset relative to UT ('+00:00')
+        params['RANGE_UNITS'] = None           # OBS: range units (AU or KM)
+        params['SUPPRESS_RANGE_RATE'] = 'NO'   # OBS: turn off output of delta-dot and rdot
+        params['ELEV_CUT']   = '-90'           # OBS: skip output when below elevation
+        params['SKIP_DAYLT'] = skip_day   # OBS: skip output when daylight
+        params['SOLAR_ELONG'] = "'0,180'"      # OBS: skip output outside range
+        params['AIRMASS']    = airmass    # OBS: skip output when airmass is > cutoff
+        params['LHA_CUTOFF'] = None            # OBS: skip output when hour angle is > cutoff
+        params['EXTRA_PREC'] = 'YES'           # OBS: show additional output digits (YES or NO)
+        params['CSV_FORMAT'] = csvformat  # Output in comma-separated value format (YES or NO)
+        params['VEC_LABELS'] = None            # label each vector component (YES or NO)
+        params['ELM_LABELS'] = None            # label each osculating element
+        params['TP_TYPE']    = None            # Time of periapsis for osculating element tables
+        params['R_T_S_ONLY'] = 'NO'            # Print only rise/transit/set (NO, TVH, GEO, RAD, YES)
+        # Skiping the section of close-approch parameters...
+        # Skiping the section of heliocentric ecliptic osculating elements...
+        response = requests.get(self.url, params=params)
+
+        return response
+
+    def get_ephemerides(self, target: Target, overwrite: bool = False) -> EphemerisCoordinates:
+ 
+        horizons_name = self._form_horizons_name(target.tag, target.des)
+        file = self._get_ephemeris_file(horizons_name)
+
+        if not overwrite and os.path.exists(file):
+            with open(file, 'r') as f:
+                lines = f.readlines()
+        
+        else:
+            res = self.query(horizons_name)
+            lines = res.text.splitlines()
+            if file != None:
+                with open(file, 'w') as f:
+                    f.write(res.text)
+
+        firstline = lines.index('$$SOE') + 1
+        lastline = lines.index('$$EOE') - 1
+        time = np.array([])
+        ra = np.array([])
+        dec = np.array([])
+
+        for line in lines[firstline:lastline]:
+            if line and line[7:15] != 'Daylight' and line[7:14] != 'Airmass':
+
+                values = line.split(' ')
+                values = d.split(' ')
```

Comment:
Not too sure what's going on here: the second command cancels out the first, since you're overwriting `values`?

Reply:
Is `d` even defined at this point? I don't see it anywhere.